travis.sh

```diff
@@ -0,0 +1,4 @@
+cargo build
+if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
+  cargo test -- --nocapture
+fi
```

Comment:
Can you embed that in `.travis.yml`? I don't think these 4 lines need to be in their own file